.origintrail_noderc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This will be mounted from the outside of the container so I don't find it necessary. Also, if we leave in on the main branch upgrade will override users config with this one so let's remove it

Reply:
Good point, we definitely should remove it, and setup to copy the file into the container

Reply:
Mount is already done here - @swamala will confirm

Reply:
This will be removed.